pysiaf/tests/test_nirspec.py

```diff
@@ -189,7 +202,8 @@ def test_nirspec_slit_transformations(verbose=False):
     """
     siaf = Siaf(instrument)
 
-    threshold = 0.1
+    threshold = 0.012  # 12 mas
+    pixel_threshold = 10 * threshold
```

Comment:
The new thresholds used in the assert statements below seem to be much more loose than before. Why did that become necessary?